docs/database/schema-alternative.sql

```diff
@@ -0,0 +1,101 @@
+--
+-- Copyright (C) 2016 Red Hat, Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--         http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+CREATE EXTENSION hstore;
+
+-- ==== Instances ====================================================================
+
+-- An action instance is the single entry point in this sub domain. It is referenced
+-- by an integration and has all configuration needed for creating an instance of
+-- a fully configured Camel connector
+CREATE TABLE action_instance
```

Comment:
You're proposing we store the action configuration (effectively Camel connector configuration) in the database. Is this duplicated in the integration project YAML? Which is authoritative?